src/main/java/net/minecraftforge/event/entity/EntityEvent.java

```diff
@@ -137,37 +137,42 @@ public EnteringChunk(Entity entity, int newChunkX, int newChunkZ, int oldChunkX,
         public int getOldChunkZ() { return oldChunkZ; }
         public void setOldChunkZ(int oldChunkZ) { this.oldChunkZ = oldChunkZ; }
     }
-    
+
     /**
-     * EyeHeight is fired when an Entity's eye height changes. <br>
      * This event is fired whenever the {@link Pose} changes, and in a few other hardcoded scenarios.<br>
+     * CAREFUL: This is also fired in the Entity constructor. Therefor the entity(subclass) might not be fully initialized.
+     * If you change the players size, you probably want to set the eye height accordingly.
      * <br>
      * This event is not {@link Cancelable}.<br>
      * <br>
      * This event does not have a result. {@link HasResult}
      * <br>
      * This event is fired on the {@link MinecraftForge#EVENT_BUS}.<br>
      **/
-    public static class EyeHeight extends EntityEvent
+    public static class Size extends EntityEvent
```

Comment:
This is a breaking change, and we don't allow them after release... probably best to just deprecate the old event and keep firing it, but also add the new event. Drop a comment to the old event like `TODO: Remove in 1.16`

Reply:
Sure, I was hoping for this to be accepted before RB, but I can change this now, if you are willing to assign it to Lex afterwards. Otherwise I will just wait for next merge conflict.

Should I create a second event or rather extend the old one with the new variables and getters/setters without changing any of the old names/signatures. Thereby, we don't have to fire two events. They could then be properly renamed in 1.16

Reply:
Not sure on the answer to your question, but when you are doing so (whichever you get told to) and deprecating the old one make sure to also include some comment/TODO about renaming/removing in 1.16 so that the forge team is able to see/remember to actually do so